features/cloud.py

```diff
@@ -189,7 +189,12 @@ def locate_image_name(self, series: str) -> str:
         elif "pro" in self.machine_type:
             image_type = ImageType.PRO
 
-        return self.api.daily_image(release=series, image_type=image_type)
+        if daily:
+            return self.api.daily_image(release=series, image_type=image_type)
+        else:
+            return self.api.released_image(
```

Comment:
Should we alert somehow that we are defaulting to a non-daily image ?

Reply:
ah good point - I'll add a log in there

Reply:
updated!